CHANGELOG.md

```diff
@@ -3,6 +3,12 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.3.6] - 2025-05-dd
+
+### `Fix`
```

Comment:
Could you add a note about including the `software_version.yml` file in IRIDA Next JSON.